.gitignore

```diff
@@ -77,12 +77,20 @@ wp-content/debug.log
 /wp-content/index.php
 /wp-content/sunrise.php
 /wp-content/Icon
+/wp-content/ewww/*
 /wp-content/plugins/*
 /wp-content/themes/*
+wp-content/hub-acq-man/*
+wp-content/hub-connector/*
+wp-content/old-hub-plugins/*
```

Comment:
This stuff would probably be better gitignored by 

```gitignore
/wp-content/*
```

and then at the bottom of the file, adding this don't-ignore from our boilerplate repo:

```gitignore
# do version-control the child theme we care about
# add a line here that looks like this:
# !/wp-content/themes/example-theme-directory
!/wp-content/themes/example-theme-directory
```

https://github.com/INN/umbrella-boilerplate/blob/84d23e76af8d9e1c611b3268b8e399c098f6d2f9/.gitignore#L119-L122

Reply:
OK! I've made these changes to .gitignore and pushed it to the repo. Do I make a new Pull Request?

Reply:
Nope; the current PR has been updated.

Reply:
Still need to add the chunk at the bottom of the .gitignore, though.

Reply:
It's on line 82!

Reply:
It's on line 79!

Reply:
Ah! So it is. My bad.